src/TodosContext/TodosContext.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../types/Todo';
+import { getTodos } from '../api/todos';
+import { USER_ID } from '../utils/USER_ID';
+import { TodosContextType } from '../types/TodosContextType';
+import { SortType } from '../types/SortType';
+
+export const TodosContext = React.createContext<TodosContextType>({
+  todos: [],
+  setTodos: () => {},
+  errorMessage: '',
+  setErrorMessage: () => {},
+  preparedTodos: [],
+  sortQuery: '',
+  setSortQuery: () => {},
+  tempTodo: {
+    id: 0,
+    userId: 0,
+    title: '',
+    completed: false,
+  },
+  setTempTodo: () => {},
+  todosInProcess: [],
+  setTodosInProcess: () => {},
+  completedTodos: [],
+});
```

Comment:
try reducing the amount of props in the component, it's bad practice to have big amount of props